.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
-- [ ] There are clear instructions on how to obtain the data
-- [ ] Tag @remi-kazeroni in this pull request, so that the new dataset can be added to the OBS data pool at DKRZ and synchronized with CEDA-Jasmin
+## Checklist
 
-Automated checks pass, status can be seen below the pull request:
+It is the responsibility of the author to make sure the PR is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.
 
-- [ ] Circle/CI tests pass. If the tests are failing, click the `Details` link to find out why.
-- [ ] Preferably Codacy code quality checks pass, however a few remaining hard to solve Codacy issues are still acceptable. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-- [ ] The documentation is building successfully on readthedocs and looks well formatted, click the `Details` link to see it.
+- [ ] [🛠][1] PR has a descriptive title for the [changelog](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review)
+- [ ] [🛠][1] Code follows the [style guide](https://docs.esmvaltool.org/en/latest/community/introduction.html#code-style)
+- [ ] [🛠][1] [Documentation](https://docs.esmvaltool.org/en/latest/community/introduction.html#documentation) is available
+- [ ] [🛠][1] YAML files pass [`pre-commit`](https://docs.esmvaltool.org/en/latest/community/introduction.html#pre-commit) or [`yamllint`](https://docs.esmvaltool.org/en/latest/community/introduction.html#yaml) checks
+- [ ] [🛠][1] [Circle/CI tests pass](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review)
+- [ ] [🛠][1] [Codacy code quality checks pass]((https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review))
+- [ ] [🛠][1] [Documentation builds successfully](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review) on readthedocs
 
+### New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
 
-**Checklist for scientific review**
+- [ ] [🛠][1] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added
+- [ ] [🛠][1] New dependencies are added to the [project requirements](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#additional-dependencies)
+- [ ] [🧪][2] [Documentation](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#documentation) for the recipe/diagnostic clearly describes how the recipe works
+- [ ] [🧪][2] Recipe runs successfully on your own machine or [`@esmvalbot`](https://github.com/apps/esmvalbot)
+- [ ] [🧪][2] Figure(s)/data [look as expected](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance)
```

Comment:
```suggestion
- [ ] [🧪][2] Figure(s)/data [look as expected](https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review)
```

Reply:
I think this is a slightly better, though still does not explain what "expected" is....

Reply:
Yeah, I also considered this one. 👍